monitoring/oplog-populator/alerts.yaml

```diff
@@ -8,6 +8,9 @@ x-inputs:
 - name: oplogPopulatorChangeStreamLagThreshold
   type: config
   value: 10
+- name: oplogPopulatorChangeStreamLimit
+  type: config
+  value: 0
```

Comment:
default value should be the value that we used in default zenko value (10), so that we get somewhat consistent default if there is some mismatch (like older zkop w/ newer backbeat)